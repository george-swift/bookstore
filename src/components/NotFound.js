import { Link } from 'react-router-dom';

export default function NotFound() {
  return (
    <div className="container">
      <div className="text-center">
        <h3>Page not Found!</h3>
        <p>
          <Link to="/"> Back to Homepage</Link>
        </p>
      </div>
    </div>
  );
}
