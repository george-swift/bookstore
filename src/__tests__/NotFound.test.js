import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NotFound from '../components/NotFound';

describe('Testing NotFound component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Router><NotFound /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
