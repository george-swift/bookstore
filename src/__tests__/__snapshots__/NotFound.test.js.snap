// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NotFound component should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="text-center"
  >
    <h3>
      Page not Found!
    </h3>
    <p>
      <a
        href="/"
        onClick={[Function]}
      >
         Back to Homepage
      </a>
    </p>
  </div>
</div>
`;
